### DISCUSSION TITLE
Que index of a specific file thru API

### DISCUSSION THREAD

**Discussion by jimmyspets** [photoprism/photoprism#4630] [Q&A]
I am considering if PhotoPrism is a good fit in a project I am working on.
The data I want to index is a large fileshare (about 100TB, 1 million photos and videos) stored on CephFS.
CephFS has a very handy feature where I can easily query for directories with updated files after a certain time, I use this often in Python for archiving etc.
Is it possible to feed PhotoPrism thru its API what files or folders should be added to be indexed so that it doesn't have to scan the entire filesystem?

**Comment by graciousgrey** [comment]
It is possible to start indexing a specific path within the originals folder using the api.
